colossalai/inference/tensor_parallel/modeling/llama.py

```diff
@@ -5,12 +5,9 @@
 from transformers.models.llama.modeling_llama import LlamaAttention, LlamaDecoderLayer, LlamaModel, LlamaRMSNorm
 
 from colossalai.inference.tensor_parallel.batch_infer_state import BatchInferState
-from colossalai.kernel.triton import (
-    copy_kv_cache_to_dest,
-    llama_context_attn_fwd,
-    rotary_embedding_fwd,
-    token_attention_fwd,
-)
+from colossalai.kernel.triton import llama_context_attn_fwd, rotary_embedding_fwd, token_attention_fwd
```

Comment:
try and except? what if the user does not have triton installed?

Reply:
The check for `triton` is done in `_init_`